docker/entrypoint.sh

```diff
@@ -25,43 +25,43 @@ case "$1" in
         if [[ "$RUN_ALL_TESTS" -eq "1" ]]; then
             # Run all the available tests for the current environment.
             # If the lightwalletd environmental variables are set, we will also run those tests.
-            cargo test --locked --release --features "$ENTRYPOINT_FEATURES" --workspace -- --nocapture --include-ignored
+            cargo test --locked --release --features "$TEST_FEATURES" --workspace -- --nocapture --include-ignored
```

Comment:
`$ENTRYPOINT_FEATURES` used to combine `$FEATURES` and `$TEST_FEATURES`. Similarly to the previous comment, will we not miss `$FEATURES`?